core/src/bundle_stage.rs

```diff
@@ -322,6 +313,19 @@ impl BundleStage {
             ),
         );
 
+        // either qos rate-limited a tx in here or bundle exceeds max cost, drop the bundle
+        let total_bundle_cost: u64 = tx_costs.iter().map(|tx_cost| tx_cost.sum()).sum();
+        if sanitized_bundle.transactions.len() != num_included
+            || total_bundle_cost > MAX_BUNDLE_COST
```

Comment:
what is MAX_BUNDLE_COST derived from?

Reply:
like where did that number come from?

Reply:
right now i just chose something bigger than was possible with 5 txs,  this was mostly just to create a parameter that can be adjusted if we increase bundle size in the future.

should i comment or just remove and worry about that when we/if increase bundle size

Reply:
imo remove so we don't need to push out another update if we hard-code value or have validators set from cli

Reply:
can limit in BE if we need to